### DISCUSSION TITLE
<bug> SVG displays incorrectly on GitHub

### DISCUSSION THREAD

**Discussion by LiamSwayne** [community/community#58335] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I uploaded this SVG several days ago:

As of today, no changes have been made to the file, but it now displays with a different font and incorrect kerning:
<img width="694" alt="Screen Shot 2023-06-17 at 4 34 27 PM" src="https://github.com/community/community/assets/108629034/ce3da8db-5855-4b27-8a12-09829ace7101">

The original font was Montserrat. Now it has been changed to a Serif font.

**Comment by Caferino** [comment]
Got a similar problem here. My SVG won't refresh or update properly, the old one keeps showing

This is what it shows in my README.md:
![Screenshot from 2024-04-24 16-11-13](https://github.com/community/community/assets/31547371/73b6766b-68af-4a9b-b2a6-90d213a90729)

This is what the SVG actually is (tiny 30x30 pixels and white, not huge and black):
![Screenshot from 2024-04-24 16-12-21](https://github.com/community/community/assets/31547371/2dcd0573-1909-4728-b8d6-1140535b1a2a)

Nothing has worked, clearing cache, hard refreh (Ctrl + Shift + R), reuploading the SVG, etc. What an insane bug, it's so frustrating

**Reply by Caferino** [reply]
Uh... well, right after posting this comment I decided to upload the same SVG in the same folder, but with a different name, and without changing the path in the README.md yet, it got fixed nonetheless. So weird. Deleted such new SVG and everything is fine now. Github either takes about 15 minutes to update SVGs, or I somehow forced it to clear/update its cache after directly inserting a new file through Github itself in a hidden folder such as .images/README, I don't know
